### DISCUSSION TITLE
Use it in Stable Diffusion?

### DISCUSSION THREAD

**Discussion by ArtaFakhari** [vladmandic/human#354] [Ideas]
Is it possible to use `human` as a `ControlNet` Preprocessor?

**Comment by vladmandic** [comment]
Not a bad idea, just need to sit down and make it play nice with each other.

**Reply by ArtaFakhari** [reply]
Perfect!
Just tell me where to start

**Comment by vladmandic** [comment]
perhaps create a feature request here so it doesn't fall off my radar? :)
